coding/cpp_style_guide.rst

```diff
@@ -767,7 +767,7 @@ Exception classes are really not part of the main design of the program, and nam
 
 .. code-block:: cpp
 
-   double & getElevation(unsigned int antennaId), void pointAntenna(Source const & source)
+   double &getElevation(unsigned int antennaId), void pointAntenna(Source const &source)
```

Comment:
Even if both are acceptable, I'd prefer that the examples attach the `&` to the type, rather than the function/variable.